### PR TITLE
support specifying the domain and port explicitly

### PR DESCRIPTION
**Pull request by vikstrous**

### CODE REVIEW

File: ambassador/run.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+if [ $1 == "/bin/sh" ]
+then
+    port=$(env | grep _TCP= | sed 's/.*_PORT_\([0-9]*\)_TCP=tcp:\/\/\(.*\):\(.*\)/\1/')
+    target=$(env | grep _TCP= | sed 's/.*_PORT_\([0-9]*\)_TCP=tcp:\/\/\(.*\):\(.*\)/\2:\3/')
+    if [ -z $target ]
+    then
+        echo 'Failed to autodetect target domain and port. Please specify then as arguments.'
```
Comment (SvenDowideit) [inline ambassador/run.sh]: I presume we should exit with an error code here :)

**Comment by SvenDowideit** [comment]
@vikstrous so by 'domain' you mean specify the source container's internal IP address?

I'm trying to work out how to document this usefully

**Comment by vikstrous** [comment]
It doesn't have to be an IP. With custom networks you can use the container's name because docker populates the hosts file if you are not on the default network.

On January 11, 2016 10:36:49 PM PST, Sven Dowideit <notifications@github.com> wrote:
>@vikstrous so by 'domain' you mean specify the source container's
>internal IP address?
>
>I'm trying to work out how to document this usefully
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/SvenDowideit/dockerfiles/pull/41#issuecomment-170813469

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by SvenDowideit** [comment]
fixed by #42

**Pull request closed without merging**